sql/dsny_mtsgaragemaintenance.sql

```diff
@@ -22,20 +22,15 @@ ALTER TABLE dsny_mtsgaragemaintenance
 
 update dsny_mtsgaragemaintenance as t
 SET hash =  md5(CAST((t.*)AS text)), 
-    the_geom = (CASE
-                    WHEN the_geom = ''
-                        THEN location
-                    ELSE the_geom
-                END),
 	facname = (CASE
-                    WHEN type = 'GARAGE' THEN CONCAT(name,' ',type)
-                    WHEN type <> 'GARAGE' THEN CONCAT(name)
+                    WHEN LOWER(type) = 'garage' THEN CONCAT(name,' ',type)
+                    WHEN LOWER(type) <> 'garage' THEN CONCAT(name)
```

Comment:
check out postgres pattern matching
https://www.postgresql.org/docs/9.3/functions-matching.html

if you do `type ~* 'garage'` it would use regex to match pattern, personally i think this looks simpler, but just my preference